README.md

```diff
@@ -1,2 +1,33 @@
 # emap-actions
-Reusable GitHub actions for EMAP
+Reusable GitHub actions for EMAP projects
+
+
+## Examples
+
+To tag a list of repositories
+```yaml
+jobs:
+  example-job:
+    uses: inform-health-informatics/emap-actions/reusable_workflows/tag_repos.yaml@main
+    with:
+      repos: "Inform-DB"    # Required, space seperated list
+      branch: "develop"     # Optional
+      tag_name: "test_me"   # Optional
+    secrets:
+      token: ${{ your-github-token }}  # Required
+```
+
+***
+
+To test a repository if a tag exists
```

Comment:
also good to mention why here, and also describe the overall workflow of combining these